nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/summary/ui/connection-status-listing/connection-status-listing.component.html

```diff
@@ -41,6 +41,9 @@
                     <div>Last updated:</div>
                     <div class="refresh-timestamp">{{ loadedTimestamp$ | async }}</div>
                 </div>
+                <div>
+                    <a (click)="openSystemDiagnostics($event)">System Diagnostics</a>
```

Comment:
There is an explicit permission for accessing system diagnostics. In current NiFi the URL is always shown and the user gets an insufficent permissions error message when clicking it. We could consider hiding the link in this case or showing the message in a dialog. Currently, it is unhandled and nothing happens in the UI. Inspecting Dev Tools shows the `403` response.

Reply:
I'll hide these links if the user does not had the appropriate permission.